confidant/services/jwkmanager.py

```diff
@@ -115,14 +137,24 @@ def get_payload(self, certificate: str, token: str,
 
         return self._payload_cache[certificate_hash][token_hash]
 
-    def get_jwks(self, key_id: str, algorithm: str = 'RS256') -> Dict[str, str]:
-        key = self._keys.get_key(key_id)
-        if key:
-            stats.incr(f'get_jwks.{key_id}.hit')
-            return {**key.export_public(as_dict=True), 'alg': algorithm}
+    def get_active_key(self, environment: str) -> jwk.JWK:
+        if environment in ACTIVE_SIGNING_KEYS and environment in self._keys:
+            return self._keys[environment].get_key(
+                ACTIVE_SIGNING_KEYS[environment]
+            )
+
+    def get_jwks(self, environment: str, algorithm: str = 'RS256') \
+            -> List[Dict[str, str]]:
+        keys = self._keys.get(environment)
+        if keys:
+            stats.incr(f'get_jwks.{environment}.hit')
+            return [{
```

Comment:
I think we want it in this format instead:


```
{
  "keys": [
    {
      "kty": "RSA",
      "e": "...",
      "n": "...",
      "alg": "RS256",
      "kid": "staging_1"
    },
{
      "kty": "RSA",
      "e": "...",
      "n": "...",
      "alg": "RS256",
      "kid": "staging_2"
    }
  ]
}
```

Reply:
Yeah this is the final format (all fields are exported by `export_public()` except `alg`). The route itself wraps the keys in the `keys` object per the schema:
https://github.com/lyft/confidant/blob/master/confidant/schema/jwks.py#L23